frontend/src/components/searchSelector/SearchSelector.tsx

```diff
@@ -39,7 +39,7 @@ type SearchSelectorProps = {
   searchHelpText?: string;
   searchPlaceholder?: string;
   searchValue: string;
-  toggleText: string;
+  toggleText: React.ReactNode | string;
```

Comment:
this same change is popping up in 3 different PRs 😆 

should be `toggleContent` as it's not limited to text now. just to make sure it doesn't get lost with rebasing all these prs

Reply:
> just to make sure it doesn't get lost with rebasing all these prs

There's just one more PR and I think we will be good to go!